packages/db/prisma/schema.prisma

```diff
@@ -653,3 +653,19 @@ model FeedbackFormAnswer {
 
   @@map("feedback_form_answer")
 }
+model AuditLog {
+  id          String   @id @default(uuid())
+
+  tableName   String
+  rowId       String?
+  createdAt   DateTime @default(now()) @db.Timestamptz(3)
+  user        User?    @relation(fields: [userId], references: [id])
+  userId      String?
+  operation   String
+  rowData     Json
+
+  // Adds transaction context
+  transactionId String?
```

Comment:
I think `BigInt` should work here. Maybe we have to do `pg_current_xact_id()::bigint` in the trigger